netlify.toml

```diff
@@ -10,7 +10,7 @@
 
 [[redirects]]
   from = "/login"
-  to = "http://preview-login.redashapp.com/login"
+  to = "https://preview-login.redashapp.com/login"
```

Comment:
I don't remember exactly why (was it necessary or just convenient), but `preview-login` doesn't support https. 

But why do you think it's causing the issue you see? It redirects to a different page. Also, as @susodapop mentioned, calling `/login` directly works for him.

Reply:
> But why do you think it's causing the issue you see? It redirects to a different page.

I think it might be a browser security restriction that Safari carries out.

> Also, as @susodapop mentioned, calling /login directly works for him.

Calling directly is ok, but redirecting from https to http might be disallowed.

I dunno. I'll look into it a bit but I don't really care 😬

Reply:
@arikfr maybe the api/session retries simply don't let the redirect happen. Are the retries and redirect coded explicitly or are they abstracted in some login mechanism?